### PR TITLE
fix build issues

### PR DESCRIPTION
**Pull request by caravinci**

### CODE REVIEW

File: components/DefaultPageLayout.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const ThreeRowPageContainer = styled(PageContainer)`
   grid-template-rows: 58px 1fr 110px;
 `;
 
-const DefaultPageLayout = ({ children, title = '' }: { children: ReactElement, title?: string }): ReactElement => {
+const DefaultPageLayout = ({ children, title = '' }: { children: ReactElement | ReactNode, title?: string }): ReactElement => {
```
Comment (ciaranschutte) [inline components/DefaultPageLayout.tsx:14]: ReactNode I think covers ReactElement? (superset?)

Comment (caravinci) [reply components/DefaultPageLayout.tsx:14]: tried that earlier, but didn't seem to cover it before. weird 🤔

File: components/pages/Home/index.tsx [hunk 2]
```diff
@@ -3,7 +3,11 @@ import React from 'react';
 import DefaultPageLayout from 'components/DefaultPageLayout';
 
 const Home = () => {
-  return <DefaultPageLayout title={'Homepage'} />;
+  return (
+    <DefaultPageLayout title={'Homepage'}>
+      &nbsp;
+    </DefaultPageLayout>
```
Comment (ciaranschutte) [inline components/pages/Home/index.tsx:9]: guessing this is just a prettier thing :)

**Review by ciaranschutte — APPROVED** [review]

File: components/pages/Home/index.tsx (hunk 2 above)
Comment (caravinci) [reply components/pages/Home/index.tsx]: no, the `children` props was required, so I added a placeholder for the home page... but instead made the children optional.

**Pull request merged**